python/setup.py

```diff
@@ -0,0 +1,5 @@
+from setuptools import setup, find_packages
+
+setup(name='ingestion',
```

Comment:
does this make sense since it's using all packages in the folder? Like if we have `python/data_serving`, what would this be?

Can you still install just `python/<some_package>` independently or now you have to install the whole thing?

Reply:
Hm, so I don't think you would be able to install just `python/<some_package>` but my thought was since we are copying the code into the docker image, if we only copy the packages we need then this would install them all properly. I think the name 'ingestion' should probably be changed to something like 'health-equity-tracker'. But now that I write it down it seems a bit hacky. WDYT?

Reply:
To make sure I'm understanding correctly, if you need `package1` but not `package2` the idea would be you copy `python/package1` but not `python/package2`, and then when you do pip install `./python` it will only install `package` because only `package` is actually there? You'd also need to manually copy over the `setup.py` file. So the dockerfile would be
```
COPY ./python/package1 ./python/package1
COPY ./python/setup.py ./python/setup.py
RUN pip install ./python // automatically installs package1 but not package2
```

Interesting idea. I think I might not be understanding the motivation. You still need to modify the Dockerfile to specify which packages to copy over. It seems like the main benefit is it makes local dev easier because you can run `pip install -e python/` just once? But I don't think it's that bothersome to do it once per directory, especially with the `-e` option.

If there's something I'm missing here let me know! I don't think this is *too* different from what we had before, but all else being equal I prefer the explicitness of the Dockerfile directly installing the deps it needs, and the isolation of having each package be separate and control its own setup. I don't have any super concrete reasons, just that it feels better to me haha. Like if someone didn't realize that the COPY command was also controlling what was being installed and just decided to change the individual COPY lines to just copy the whole `./python` package, and then it breaks because it's missing external deps.

Reply:
Yeah I think you're right. There could be some merit still in only copying over the packages we need but having the setup.py file at the top level is more confusing than helpful. I moved setup.py back into the ingestion/ folder and reverted the Docker files.

Reply:
Yeah definitely valid to just copy over the relevant package code, that's an easy 1-line change to the dockerfile and doesn't affect package structure, you're welcome to do that if you think that's better